src/Pages/AboutUs/Aboutus.js

```diff
@@ -52,46 +51,42 @@ export default function Aboutus() {
             of them:
           </p>
           <div className="company-container">
-            <div className="company1"><img src={DeloitteLogo} /></div>
-            <div className="company2"><img src={MckinseyLogo} /></div>
-
+            <div className="company1">
+              <img src={DeloitteLogo} />
+            </div>
+            <div className="company2">
+              <img src={MckinseyLogo} />
+            </div>
           </div>
           <div className="flex-row">
-          <Fade>
-            <div className="seeking-partnership-with-us">
-              <h3>Seeking partnership with us?</h3>
-            </div>
-            <p className="paragraphs-3">
-              Western dev society plans on holding both in person and online
-              events for the upcoming 2022-2023 school year. Running live events
-              for the WDS and Western communities comes at a very high cost.
-              Therefore, WDS highly appreaciates and welcomes any financial or
-              in-kind sponsorships
-              <br />
-              <br />
-              There are many benefits that come with collaborating with our
-              association! Partnering with WDS means that we will share and
-              promote your organization&#x27;s marketing material to the Western
-              Engineering &amp; Computer Science and greater Western community.
-              You&#x27;ll also have opportunities to directly engage with the
-              student population through sending representatives from your
-              company, and/or through other collaboration formats. This is great
-              for recruitment purposes!
-            </p>
+            <Fade>
+              <div className="seeking-partnership-with-us">
+                <h3>Seeking partnership with us?</h3>
+              </div>
+              <p className="paragraphs-3">
+                Western dev society plans on holding both in person and online
+                events for the upcoming 2022-2023 school year. Running live
+                events for the WDS and Western communities comes at a very high
+                cost. Therefore, WDS highly appreaciates and welcomes any
+                financial or in-kind sponsorships
+                <br />
+                <br />
+                There are many benefits that come with collaborating with our
+                association! Partnering with WDS means that we will share and
+                promote your organization&#x27;s marketing material to the
+                Western Engineering &amp; Computer Science and greater Western
+                community. You&#x27;ll also have opportunities to directly
+                engage with the student population through sending
+                representatives from your company, and/or through other
+                collaboration formats. This is great for recruitment purposes!
+              </p>
```

Comment:
Text this long should be stored in a separate file, not within the HTML itself

Reply:
agreed, but for now i've just stored it in a constant in the same file. felt weird making a whole file just for that one constant, and doesn't rlly fit with members data yk.